go.mod

```diff
@@ -15,7 +15,15 @@ require (
 	github.com/giantswarm/versionbundle v0.2.0
 	github.com/prometheus/client_golang v1.9.0
 	github.com/spf13/viper v1.7.1
-	k8s.io/api v0.17.2
-	k8s.io/apimachinery v0.17.2
-	k8s.io/client-go v0.17.2
+	k8s.io/api v0.18.6
+	k8s.io/apimachinery v0.18.6
+	k8s.io/client-go v0.18.6
```

Comment:
These were automatically updated after running `go mod tidy`

Reply:
Can we just revert all of the `go.mod` changes? We weren't planning to update this operator to kubernetes 1.18 client libraries.

Reply:
Yes, reverted and CI is green :)